### PR TITLE
Fixes item 2 of #2106

### PR DESCRIPTION
**Pull request by Mewchild**
Changes access on some security doors from access_security (1) to access_sec_doors (63). This makes it so IAA can enter security, as well as fixes guest passes for the security common areas. Previously, letting someone into security with a pass meant you had to give them access to the green armory as well.

#2106

### CODE REVIEW

**Comment by Mewchild** [comment]
I am a simply man. I{ saw an open issue, I fix open issue. Given this only allows access tot he sec hallway, briefing room, and break room, is it an issue? I know most heads have access to most common areas, with the HoS not being allowed into medical as a major exception. Should heads not have basic access to departments outside their own?

**Comment by Arokha** [comment]
The 'trade' that was achieved was that the HoS stops having access to medical, and that everyone else stops having access to security.

Does IAA not have any unique access you can use instead?

**Comment by Mewchild** [comment]
I'll fix it up. IAA access is 38, I'll play with this and try to remember exactly how req_access_one works

**Comment by Arokha** [comment]
Perhaps a better solution would be to remove the access_sec_doors from all the head accesses. Not an amazing solution, but perhaps more sensible.

File: code/game/jobs/job/engineering.dm [hunk 1]
```diff
@@ -19,11 +19,11 @@
 
 	access = list(access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels,
 			            access_teleporter, access_external_airlocks, access_atmospherics, access_emergency_storage, access_eva,
-			            access_heads, access_construction, access_sec_doors,
+			            access_heads, access_construction,
```
Comment (Arokha) [inline code/game/jobs/job/engineering.dm]: //VOREStation Edit

File: code/game/jobs/job/engineering.dm [hunk 2]
```diff
@@ -19,11 +19,11 @@
 
 	access = list(access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels,
 			            access_teleporter, access_external_airlocks, access_atmospherics, access_emergency_storage, access_eva,
-			            access_heads, access_construction, access_sec_doors,
+			            access_heads, access_construction,
 			            access_ce, access_RC_announce, access_keycard_auth, access_tcomsat, access_ai_upload)
 	minimal_access = list(access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels,
 			            access_teleporter, access_external_airlocks, access_atmospherics, access_emergency_storage, access_eva,
-			            access_heads, access_construction, access_sec_doors,
+			            access_heads, access_construction,
```
Comment (Arokha) [inline code/game/jobs/job/engineering.dm]: //VOREStation Edit

File: code/game/jobs/job/medical.dm [hunk 3]
```diff
@@ -14,10 +14,10 @@
 	economic_modifier = 10
 	access = list(access_medical, access_medical_equip, access_morgue, access_genetics, access_heads,
 			access_chemistry, access_virology, access_cmo, access_surgery, access_RC_announce,
-			access_keycard_auth, access_sec_doors, access_psychiatrist, access_eva, access_external_airlocks, access_maint_tunnels)
+			access_keycard_auth, access_psychiatrist, access_eva, access_external_airlocks, access_maint_tunnels)
```
Comment (Arokha) [inline code/game/jobs/job/medical.dm]: Etc

**Comment by Mewchild** [comment]
HoP will still have access_sec_doors, as the HoP is often the acting director, and should have basic access to every department, I feel.

**Comment by Arokha** [comment]
@Leshana halp

**Comment by Mewchild** [comment]
Closing for now, until #3534 is completed and merged

**Pull request closed without merging**